sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -316,17 +329,17 @@ export class EducationProgram extends RecordDataModel {
     type: "timestamptz",
     nullable: false,
   })
-  submittedOn: Date;
+  submittedDate: Date;
 
   /**
-   * Education program status updated on.
+   * Education program assessed date.
    */
   @Column({
-    name: "status_updated_on",
+    name: "assessed_date",
     type: "timestamptz",
     nullable: false,
   })
-  statusUpdatedOn: Date;
+  assessedDate: Date;
```

Comment:
Please change this to a nullable and change also the decorator property.

Reply:
Actually it is not suppose to be. It was renamed as assessed_date from status_updated_by and that column was mandatory. I can add a db migration to make it nullable and change the entity.

Reply:
statusUpdatedOn was not supposed to be optional because every time there will be a status present.
AssessedData is not supposed to be populated upon submission, only if automatically approved.